packages/components/src/index.ts

```diff
@@ -10,7 +10,9 @@ export * from './Link';
 export * from './Modal';
 export * from './PopupTrigger';
 export * from './Select';
-export * from './Spinner';
+export * from './spinners/Spinner';
+export * from './spinners/CenteredSpinner';
```

Comment:
I'm not thrilled about promoting CenteredSpinner to a package. I think all uses of this could probably be replaced by SpinnerOverlay.

Reply:
It's quite likely there are a number of legit uses of an "inline" spinner where a full blown overlay is overkill, but yeah.... offer it for now, revisit later if needed.

Reply:
> a number of legit uses of an "inline" spinner 

Yes, but that's `Spinner`, not `CenteredSpinner`.

I think `CenteredSpinner` is effectively an overlay that only works when there's no content to be overlaid on top of.